src/components/addMovie/AddMovie.jsx

```diff
@@ -1,70 +1,130 @@
-import React, { useState } from 'react';
+import React from 'react';
 import Form from 'react-bootstrap/Form';
 import Button from 'react-bootstrap/Button';
 import { genresOptions, labels, text } from '../../data.js';
 import { useDispatch } from 'react-redux';
 import { createMovie } from 'service/index.js';
+import { Formik } from "formik";
+import * as yup from "yup";
+import { FormTextField, FormSelectField } from "components/form-field";
 
 const AddMovie = ({ handleClose }) => {
     const { titleLabel, releaseDateLabel, selectDateLabel, movieUrlLabel, genreLabel, selectGenreLabel, overviewLabel, runtimeLabel, hereLabel } = labels;
     const { resetTxt, submitTxt } = text;
     
-    const [title, setTitle] = useState("");
-    const [release_date, setReleaseDate] = useState("");
-    const [poster_path, setMovieUrl] = useState("");
-    const [genres, setGenre] = useState([]);
-    const [overview, setOverview] = useState("");
-    const [runtime, setRuntime] = useState(0);
-
     const dispatch = useDispatch();
 
     const createMovieFunc = (params) => {
         createMovie(dispatch, params, () => {
-            handleClose()
+            handleClose();
         })
     }
 
+    const schema = yup.object({
```

Comment:
Minor thing but in future you'll have many kinds of schemas, and here the name of this one is too general, needs to be more specific, e.g. movieDataSchema